lib/wpcom-error-handler/wpcom-error-handler.php

```diff
@@ -77,12 +84,19 @@ function wpcom_get_error_backtrace( $last_error_file, $last_error_type, $for_irc
 		$call_path[] = trim( $path );
 	}
 
-	return implode( ', ', $call_path );
+	return join( ', ', $call_path );
```

Comment:
We use `join()` everywhere, replacing `implode()` with `join()` for consistency.

Reply:
The canonical name of the function is `implode()` (opposite of `explode()`). The `join()` is just an alias, and PHP very occasionally does remove aliases. I'd rather see all instances of `join()` changed to `implode()`.

Reply:
Reverted in 989b3dc93